### ISSUE TITLE
Kapre with Keras 2.3.1 issues

### ISSUE THREAD

**Issue by XReyRobert** [keunwoochoi/kapre#61]
For the records, while trying to debug some issues I had with a new environment I realised that my kapre melspectrogram were wrong when using **keras 2.3.1** (latest) and 2.3.0

Reverting to **keras 2.2.5** (like my stable environment) solved the Issue. (Keras 2.2.4 also good)

parameters were as this:
```
fmin = 70   
fmax = 496
hop_length = 512
n_fft = 4096 
n_mels =  110

model = Sequential()
model.add(Melspectrogram(sr=44100, n_mels=n_mels, fmin=fmin, fmax=fmax,
  n_dft=n_fft, n_hop=hop_length, input_shape=input_shape, power_melgram=1,
  return_decibel_melgram=True, 
  trainable_kernel=True, name='melgram'))  
```  

With Keras 2.2.5
![Screenshot 2019-11-26 at 10 51 37](https://user-images.githubusercontent.com/38261891/69618833-d7081480-103a-11ea-9ccf-0f28800523ee.png)

With Keras 2.3.1
![Screenshot 2019-11-26 at 10 49 38](https://user-images.githubusercontent.com/38261891/69618908-fbfc8780-103a-11ea-9c1c-9763cbff8407.png)

Now It looks a lot like the issue I encountered trying to use #58 & #56 with the embeded tensorflow.keras (issues tested with tensorflow 2 and latest tensorflow 1.15)
BUT the keras version embedded with tensorflow 1.15.0 is supposed to be **keras 2.2.4-tf** and keras 2.2.4 standalone didn't display this issue.

**Comment by XReyRobert** [comment]
Check this Colab to quickly reproduce the issue

https://gist.github.com/XReyRobert/017b03712e367c1761e4e0f955ca3787

**Comment by keunwoochoi** [comment]
Hi, thanks for reporting it. To be honest I don't really have any bandwidth to dig in this issue and ended up responding back with such a delay without any concrete idea. Do you have any idea to fix this problem to the up-to-date tensorflow?

**Comment by XReyRobert** [comment]
Hi keunwoo, no I'm sorry, I haven't looked into the details of your implementation so I have no idea about what could be causing this...

**Comment by keunwoochoi** [comment]
Nothing was directly related to this issue, but maybe today's update could've fixed it? `pip install kapre==0.1.7` and try!
